internals/secrethub/org_init.go

```diff
@@ -39,14 +39,30 @@ func (cmd *OrgInitCommand) Register(r Registerer) {
 func (cmd *OrgInitCommand) Run() error {
 	var err error
 
+	client, err := cmd.newClient()
+	if err != nil {
+		return errio.Error(err)
+	}
+
+	user, err := client.Me().GetUser()
+	if user.Username == "Replace this predicate with a check whether user is on free plan" {
+		wantsTrial, err := ui.AskYesNo(cmd.io, "Creating an organization is not available on the free plan, would you to like to proceed and start a 14-day trial?", ui.DefaultNo)
+		if err != nil {
+			return errio.Error(err)
+		}
+		if !wantsTrial {
+			return nil
+		}
+	}
+
 	incompleteInput := cmd.name == "" || cmd.description == ""
 	if cmd.force && incompleteInput {
 		return ErrMissingFlags
 
 	} else if !cmd.force && incompleteInput {
 		fmt.Fprintf(
 			cmd.io.Stdout(),
-			"Before initializing a new organization, I need to know a few things about your organization. "+
+			"Before initializing a new organization, we need to know a few things about your organization. "+
```

Comment:
:+1:

Maybe rephrase it even more. This information is mostly for yourself and for the other users in the organization, not for we/us.